### PR TITLE
Added landing page html file and css file

### PR DESCRIPTION
**Pull request by poojavishnoi**
This is my Component Library's landing page

### CODE REVIEW

File: index.css [hunk 1]
```diff
@@ -0,0 +1,147 @@
+@import url("https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Bebas+Neue&family=IBM+Plex+Serif:wght@300&family=Lora&family=Merriweather:wght@300&family=Montserrat:wght@300&family=Roboto:wght@300&display=swap");
+
+.font {
```
Comment (sonishreyas) [inline index.css:3]: You can remove the other fonts as only the last one is used

File: index.css [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+  font-family: "Roboto", sans-serif;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+.header {
```
Comment (sonishreyas) [inline index.css:13]: define all the colors inside root-styles so it will be easy for you to reference it elsewhere

File: index.html [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+          <p>
+            NeoDev UI is a Component Library that gives you the building block
+            you need to build your own application
+          </p>
+          <button class="font">Get Started</button>
+        </div>
+        <img class="main_image" src="./images/doc.svg" alt=""/>
+      </div>
+      <div class="footer">
+        <img src="./images/github-sign.png" alt="" />
```
Comment (sonishreyas) [inline index.html:33]: You can use fontawesome images for this so you can change the images color according to your theme

File: index.html [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    <link rel="stylesheet" href="index.css">
+  </head>
+
+  <body>
+    <div id="app" class="font">
+      <div class="header">
+        <div class="header_logo">
+          <a href="#"><h1>NeoDev UI</h1></a>
+        </div>
+        <div class="header_links">
```
Comment (sonishreyas) [inline index.html]: This is just a suggestion but you can name the div class to be header_links_container so it can be differentiated from header_links

**Review by sonishreyas — COMMENTED** [review]
This looks good. There are few suggestions I have kept below. You can take a look at them.
Also, you have added a media query but when I look at it in a mobile screen display it doesn't fit the viewport. You can debug that once.

**Comment by ravikumar0403** [comment]
looks good and adding to shreyas' suggestions. 

- Use css variables for colors. 
- You can place the landing page content in column layout on smaller screen for better readability

**Review by ravikumar0403 — COMMENTED** [review]
looks good and adding to shreyas' suggestions.

Use css variables for colors.
You can place the landing page content in column layout on smaller screen for better readability

File: index.html (hunk 3 above)
Comment (poojavishnoi) [reply index.html:33]: Great suggestion!

File: index.css (hunk 1 above)
Comment (poojavishnoi) [reply index.css:3]: I didn't know this..Thanks!!

File: index.css (hunk 2 above)
Comment (poojavishnoi) [reply index.css:13]: Okay.Will do this.

File: index.html (hunk 4 above)
Comment (poojavishnoi) [reply index.html]: Okay.

**Pull request merged**